File: Course-3/Mobile-dev/Task_1/Base/app/src/main/java/org/hse/android/MainActivity.java
package org.hse.android;

import androidx.appcompat.app.AppCompatActivity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import java.util.Date;

public class MainActivity extends AppCompatActivity {

    public Date currentTime;
    private static final String TAG = "MainActivity";

    @Override protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        //getSupportActionBar().hide();

        View button_student = findViewById(R.id.button_student);
        View button_teacher = findViewById(R.id.button_teacher);

        button_student.setOnClickListener(new View.OnClickListener() {
            @Override public void onClick(View v) {
                //Intent intent = new Intent(MainActivity.this, StudentActivity.class);
                //startActivity(intent);
                Student_pressed();
            }
        });
        button_teacher.setOnClickListener(new View.OnClickListener() {
            @Override public void onClick(View v) {
                //Intent intent = new Intent(MainActivity.this, TeacherActivity.class);
                //startActivity(intent);
                Teacher_pressed();
            }
        });
    }
    private void Student_pressed(){
        Intent intent = new Intent(MainActivity.this, StudentActivity.class);
        startActivity(intent);
    }
    private void Teacher_pressed(){
        Intent intent = new Intent(MainActivity.this, TeacherActivity.class);
        startActivity(intent);
    }
}

File: Course-3/Mobile-dev/Stepik/4_ui/4-4/4-4-8/app/src/main/java/com/example/company/myapplication/MainActivity.kt
package com.example.company.myapplication

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var i: Int = 0
        textView.setOnClickListener { i++
            textView.text = "${i}"
        }

    }
}
